### PR TITLE
Let the client use a different bintray repo name

### PR DESCRIPTION
**Pull request by jgsogo**
Get the name for user repo from environment variable `BINTRAY_REPOSITORY` or fallback to `public-conan`.

### CODE REVIEW

File: bincrafters/build_shared.py [hunk 1]
```diff
@@ -104,7 +105,7 @@ def get_conan_remotes(username):
     # require other packages from the bincrafters repo.
     bincrafters_user = "bincrafters"
     if username != bincrafters_user:
-        remotes.append(get_user_repository(bincrafters_user))
+        remotes.append(get_user_repository(bincrafters_user, "public-conan"))
```
Comment (uilianries) [inline bincrafters/build_shared.py]: You could replace this line by:

**Comment by jgsogo** [comment]
ETA to next release in pypi? I need it for my jobs to upload the packages.

**Comment by uilianries** [comment]
10 min from now

**Comment by uilianries** [comment]
@jgsogo Release 0.12.0 is done!

https://pypi.python.org/pypi/bincrafters-package-tools/0.12.0

Please, update.

**Comment by jgsogo** [comment]
Great!! Thanks!!

**Pull request merged**